app/javascript/packages/document-capture/components/selfie-capture.jsx

```diff
@@ -204,15 +217,48 @@ function SelfieCapture({ value, onChange, errorMessage, className }, ref) {
                 </button>
               </>
             ) : (
-              <div className="selfie-capture__consent-prompt">
-                <strong className="selfie-capture__consent-prompt-banner usa-file-input__banner-text">
-                  {t('doc_auth.instructions.document_capture_selfie_consent_banner')}
-                </strong>
-                <p>{t('doc_auth.instructions.document_capture_selfie_consent_reason')}</p>
-                {isAccessRejected && (
-                  <p>{t('doc_auth.instructions.document_capture_selfie_consent_blocked')}</p>
-                )}
-              </div>
+              <>
+                <div className="selfie-capture__consent-prompt">
+                  <strong className="selfie-capture__consent-prompt-banner usa-file-input__banner-text">
+                    {t('doc_auth.instructions.document_capture_selfie_consent_banner')}
+                  </strong>
+                  {!isAccessRejected && (
```

Comment:
would it make sense to ternary this + the next stanza since they are mutually exclusive?

```
isAccessRejected ?
  (<...>) :
  (<...>)
```

Reply:
> would it make sense to ternary this + the next stanza since they are mutually exclusive?

Good idea! 2bd12e498